labelbox/orm/model.py

```diff
@@ -239,11 +239,58 @@ class EntityMeta(type):
     of the Entity class object so they can be referenced for example like:
         Entity.Project.
     """
+    mappings: Dict[str, "Entity"] = {}
```

Comment:
nit: this code is pretty complicated. Could we rename `mappings` so that it tells more of what it's used for?